From: Thomas Hamel <thomas-externe.hamel@edf.fr>
Date: Wed, 24 Jan 2018 15:52:19 +0100
Subject: node-mark-offline: support maint* state for slurm

When there is an active maintainance reservation, the node is
maint* instead of idle. This restore normal behavior for tests
during the maintainance.
---
 helpers/node-mark-offline | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helpers/node-mark-offline b/helpers/node-mark-offline
index b053c89..d709bdf 100644
--- a/helpers/node-mark-offline
+++ b/helpers/node-mark-offline
@@ -65,7 +65,7 @@ elif [[ "$NHC_RM" == "slurm" ]]; then
     OLD_NOTE_LEADER="${LINE[1]}"
     OLD_NOTE="${LINE[*]:2}"
     case "$STATUS" in
-        alloc*|comp*|drain*|drng*|fail*|idle*|mix*|resume*|undrain*)
+        alloc*|comp*|drain*|drng*|fail*|idle*|mix*|resume*|undrain*|maint*)
             case "$STATUS" in
                 drain*|drng*|fail*)
                     # If the node is already offline, and there is no old note, and
